packages/cli/package.json

```diff
@@ -35,6 +35,7 @@
     "yeoman-test": "^1.7.0"
   },
   "dependencies": {
+    "@types/util.promisify": "^1.0.0",
```

Comment:
This shouldn't be needed, CLI is written in vanilla JavaScript.

See also our existing [packages/core/src/promisify.ts](https://github.com/strongloop/loopback-next/blob/d9067147a1c23d47228b911eb41cd2adc71fe129/packages/core/src/promisify.ts).